underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -213,17 +216,18 @@ public FileSystem load(String userKey) throws Exception {
       Object o = c.newInstance(URI.create(ufsUri.toString()), hdfsConf, mUfsConf);
       if (o instanceof HdfsActiveSyncProvider) {
         hdfsActiveSyncProvider = (HdfsActiveSyncProvider) o;
-        LOG.info("Successfully instantiated SupportedHdfsActiveSyncProvider");
+        LOG.info("Successfully instantiated {}", SupportedHdfsActiveSyncProvider.class.getCanonicalName());
```

Comment:
line too long which make the style check failed

Reply:
Thanks.